Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md

```diff
@@ -0,0 +1,946 @@
+| FireEyeHX.Acquisitions.Files.md5 | Unknown | File md5. | 
+| FireEyeHX.Acquisitions.Files.req_filename | Unknown | The file name. | 
+| FireEyeHX.Acquisitions.Files.req_path | Unknown | The file path. | 
+| FireEyeHX.Acquisitions.Files.host._id | Unknown | FireEye HX agent ID. | 
+
+### fireeye-hx-create-indicator
+***
+Create new indicator
+
+
+#### Base Command
+
+`fireeye-hx-create-indicator`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| category | The indicator category. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| FireEyeHX.Indicators.active_since | date | Date indicator became active. | 
+| FireEyeHX.Indicators.meta | string | Meta data for new indicator | 
+| FireEyeHX.Indicators.display_name | string | The indicator display name | 
+| FireEyeHX.Indicators.name | string | The indicator name as displayed in the UI. | 
+| FireEyeHX.Indicators.created_by | string | The "Created By" field as displayed in UI | 
+| FireEyeHX.Indicators.url | string | The data URL | 
+| FireEyeHX.Indicators.create_text | Unknown | The indicator create text | 
+| FireEyeHX.Indicators.platforms | string | List of families of operating systems. | 
+| FireEyeHX.Indicators.create_actor._id | number | The ID of the actor | 
+| FireEyeHX.Indicators.create_actor.username | string | Actor user name | 
+| FireEyeHX.Indicators.signature | string | Signature of indicator  | 
+| FireEyeHX.Indicators._revision | string | Indicator revision | 
+| FireEyeHX.Indicators._id | string | FireEye unique indicator ID. | 
+| FireEyeHX.Indicator.description | string | Indicator description | 
+| FireEyeHX.Indicators.category._id | number | Category ID | 
+| FireEyeHX.Indicators.category.name | string | Category name | 
+| FireEyeHX.Indicators.category.share_mode | string | Category share mode | 
+| FireEyeHX.Indicators.category.uri_name | string | Category uri name | 
+| FireEyeHX.Indicators.category.url | string | Category URL | 
+| FireEyeHX.Indicators.uri_name | string | The indicator uri name | 
+| FireEyeHX.Indicators.stats.active_conditions | number | Indicator active conditions | 
+| FireEyeHX.Indicators.stats.alerted_agents | number | Total number of agents with HX alerts associated with this indicator. | 
+| FireEyeHX.Indicators.stats.source_alerts | number | Total number of source alerts associated with this indicator. | 
+| FireEyeHX.Indicators.update_actor._id | number | Update actor ID | 
+| FireEyeHX.Indicators.update_actor.username | string | Update actor name | 
+
+### fireeye-hx-delete-host-set-policy
+***
+Delete a Host Set Policy.
+
+
+#### Base Command
+
+`fireeye-hx-delete-host-set-policy`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| hostSetId | The Host Set Id. | Required | 
+| policyId | The Policy Id. | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
```

Comment:
Did you run the generater readme with command examples? If not please do. Consult with Jasmine on how to do it.